analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -30,10 +30,14 @@ internal class SETestContext
     {
         public readonly ValidatorTestCheck Validator = new();
 
-        public SETestContext(string code, AnalyzerLanguage language, SymbolicCheck[] additionalChecks)
+        public SETestContext(string code, AnalyzerLanguage language, SymbolicCheck[] additionalChecks, string localFunctionName = null)
         {
             const string Separator = "----------";
             var cfg = TestHelper.CompileCfg(code, language);
+            if (localFunctionName != null)
+            {
+                cfg = cfg.GetLocalFunctionControlFlowGraph(cfg.LocalFunctions.Single(x => x.Name == localFunctionName));
+            }
```

Comment:
Let's move this inside `CompileCfg`. `RoslynLiveVariableAnalysisTest.cs` uses it also right after calling `CompileCfg`.